.mocharc.cjs

```diff
@@ -5,7 +5,7 @@ module.exports = {
   timeout: 30000,
   ui: 'bdd',
   extension: ['js', 'spec.cjs', 'spec.mjs', 'spec.ts', 'spec.mts'],
-  require: ['mocha-expect-snapshot/old-format', 'test/mocha.config.cjs'],
+  require: ['mocha-expect-snapshot/old-format', 'test/mocha.config.mjs'],
```

Comment:
It’s not possible to require an esm (mjs) module. It must be cjs.

Reply:
fixed on the new, non reverted PR

Reply:
sorry but updating snapshots will not fix the dependabot problem.